### ISSUE TITLE
[plugin:vite:import-analysis] Failed to resolve import

### ISSUE THREAD

**Issue by burakakca** [DeFUCC/gun-vue#14]
Hello ,

When i click the chat button i get this error;
![image](https://user-images.githubusercontent.com/56919004/158622600-576736d0-a1c7-4115-82a2-1753c689d707.png)

Problem because of  some file first letter big or small.

I fix it for you .

```js
import "virtual:windi.css";
import "./styles/index.css";
import "./styles/transitions.css";

// ACCOUNT

export { default as AccountProfile } from "./account/profile.vue";
export { default as AccountAvatar } from "./account/avatar.vue";
export { default as AccountBadge } from "./account/badge.vue";
export { default as AccountHome } from "./account/home.vue";
export { default as AccountStars } from "./account/stars.vue";
export { default as AccountReactions } from "./account/Reactions.vue";

// LINKs

export { default as AccountMateButton } from "./account/mate/Button.vue";
export { default as AccountMateList } from "./account/mate/List.vue";

// USER

export { default as UserCredentials } from "./user/credentials.vue";
export { default as UserPassphrase } from "./user/Pass.vue";
export { default as UserAuth } from "./user/auth.vue";
export { default as UserLogin } from "./user/login.vue";
export { default as UserHome } from "./user/home.vue";
export { default as UserIcon } from "./user/icon.vue";
export { default as UserPanel } from "./user/panel.vue";
export { default as UserProfile } from "./user/profile.vue";
export { default as UserProfileField } from "./user/profile/field.vue";

export { default as RoomProfile } from "./room/profile.vue";

//SPACE

export { default as SpacePlane } from "./space/plane.vue";
export { default as SpaceGuest } from "./space/guest.vue";
export { default as SpaceArrow } from "./space/Arrow.vue";

// POSTS

export { default as PostCard } from "./post/card.vue";
export { default as PostForm } from "./post/form.vue";
export { default as PostList } from "./post/list.vue";
export { default as PostPage } from "./post/page.vue";

export { default as PostFeedBlock } from "./post/feed/Block.vue";
export { default as PostFeedLabel } from "./post/feed/Label.vue";
export { default as PostFeedList } from "./post/feed/List.vue";

export { default as PostActionStar } from "./post/action/Star.vue";
export { default as PostActionUpdate } from "./post/action/Update.vue";
export { default as PostActionBan } from "./post/action/Ban.vue";

//CHAT

export { default as ChatRoom } from "./chat/Room.vue";
export { default as ChatMessage } from "./chat/Message.vue";
export { default as ChatPrivate } from "./chat/Private.vue";
export { default as ChatPrivateCount } from "./chat/PrivateCount.vue";

// FORMS

export { default as FormLink } from "./form/Link.vue";
export { default as FormPicture } from "./form/Picture.vue";
export { default as FormText } from "./form/Text.vue";
export { default as FormYoutube } from "./form/Youtube.vue";

// UTILS

export { default as LogTree } from "./log/tree.vue";

export { default as QrShow } from "./qr/show.vue";
export { default as QrLoad } from "./qr/load.vue";

export { default as EmbedYoutube } from "./embed/youtube.vue";

export { default as UiModal } from "./ui/modal.vue";

export { default as UtilPulse } from "./util/pulse.vue";
export { default as UtilGraph } from "./util/graph.vue";
export { default as UtilShare } from "./util/share.vue";
export { default as Utiltools } from "./util/tools.vue";
export { default as UtilRelay } from "./util/Relay.vue";

export * from "@composables";
```

**Comment by burakakca** [comment]
OS : Linux Mint 20.2 Cinnamon

**Comment by lospringliu** [comment]
`git mv` to change the capitalization in git repos

**Comment by scenaristeur** [comment]
same kind of thing for me on local when clicking on Topics icon

![Capture d’écran du 2022-05-18 00-24-54](https://user-images.githubusercontent.com/4020744/168922485-1dd449a9-c391-4f67-a4a9-679679fd253a.png)

**Comment by scenaristeur** [comment]
Vue seems to use PascalCase : necessarily two words with Capital

https://eslint.vuejs.org/rules/component-name-in-template-casing.html

**Comment by davay42** [comment]
Looks like something goes wrong at [unplugin-vue-components ](https://github.com/antfu/unplugin-vue-components) in vite build. May be some node parameters need to be set? 
I'm considering new ways to structure the components, need some time to figure it out. 🤓

**Comment by davay42** [comment]
Check if these parameters are in your `vite.config.js`. 
```js
Components({
  ...
  deep: true,
  directoryAsNamespace: true,
  ...
})
```

**Comment by scenaristeur** [comment]
same warnings 
![image](https://user-images.githubusercontent.com/4020744/169784878-9d7f58f6-14ce-40e0-9380-080796297067.png)

**Comment by davay42** [comment]
So there's some Linux file system feature to deal with. It's strange that nav-bar loads correctly though.

**Comment by davay42** [comment]
@scenaristeur @burakakca Guys! I think I've fixed this issue with full rearrangement of the components in 3b47c98ac103c46a39aa8437a3f199f2593ccb6c

I've turned the directory-as-namespace option for the `unplugin-vue-components` plugin and renamed all the components to their full names. They are imported in `index.js` files in each folder and then reexported from the root `index.js`. Hope this should work for you.

P.S. Now there's also a component stories showcase enabled by Histoire [https://gun-vue.js.org/components/) - I'll try to bring all of the components there for better documentation and examples. Stay tuned!
